ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/MigrationsTest.java

```diff
@@ -274,15 +333,44 @@ private static void verifyMigrationsApplied() {
 
     // verify foo
     final List<StreamedRow> foo = assertThatEventually(
-        () -> makeKsqlQuery("SELECT * FROM FOO EMIT CHANGES LIMIT 2;"),
-        hasSize(4)); // first row is a header, last row is a message saying "Limit Reached"
+        () -> makeKsqlQuery("SELECT * FROM FOO EMIT CHANGES LIMIT 4;"),
+        hasSize(6)); // first row is a header, last row is a message saying "Limit Reached"
+    assertThat(foo.get(1).getRow().get().getColumns().size(), is(3));
     assertThat(foo.get(1).getRow().get().getColumns().get(0), is("HELLO"));
-    assertThat(foo.get(2).getRow().get().getColumns().get(0), is("GOODBYE"));
+    assertThat(foo.get(1).getRow().get().getColumns().get(1), is(50));
+    assertThat(foo.get(1).getRow().get().getColumns().get(2), is(-4325));
+    assertThat(foo.get(2).getRow().get().getColumns().get(0), is("GOOD'BYE"));
+    assertNull(foo.get(2).getRow().get().getColumns().get(1));
+    assertNull(foo.get(2).getRow().get().getColumns().get(2));
+    assertThat(foo.get(3).getRow().get().getColumns().get(0), is("mua--ha\nha"));
+    assertThat(foo.get(4).getRow().get().getColumns().get(0), is(""));
+
+    // verify bar
+    final List<StreamedRow> bar = assertThatEventually(
+        () -> makeKsqlQuery("SELECT * FROM `bar` EMIT CHANGES LIMIT 4;"),
+        hasSize(6)); // first row is a header, last row is a message saying "Limit Reached"
+    assertThat(bar.get(1).getRow().get().getColumns().get(0), is("HELLOwoo'hoo"));
+    assertThat(bar.get(2).getRow().get().getColumns().get(0), is("GOOD'BYEwoo'hoo"));
+    assertThat(bar.get(3).getRow().get().getColumns().get(0), is("mua--ha\nhawoo'hoo"));
+    assertThat(bar.get(4).getRow().get().getColumns().get(0), is("woo'hoo"));
+
+    verifyConnector("C", true);
```

Comment:
This method needs to be updated to account for async execution, right?
```suggestion
    assertThatEventually(() -> makeKsqlRequest("SHOW CONNECTORS;"), hasSize(1));
```